tox.ini

```diff
@@ -3,12 +3,25 @@ envlist = {py33, py34, py35}-{tests, tests-negtz, style}
 skip_missing_interpreters = True
 
 [testenv]
-passenv = LANG
+passenv =
+  LANG
+  CI
+  TRAVIS
+  TRAVIS_BRANCH
+  TRAVIS_BUILD_ID
+  TRAVIS_COMMIT
+  TRAVIS_JOB_ID
+  TRAVIS_JOB_NUMBER
+  TRAVIS_PULL_REQUEST
+  TRAVIS_REPO_SLUG
 deps =
+    codecov
     pytest
+    pytest-cov
     pytest-capturelog
 commands =
-	py.test {posargs}
+    py.test --cov khal {posargs}
+    codecov -e TOXENV
```

Comment:
No, it doesn't upload anything since the upload token is undefined (see the travis-specific variables passed above).
It exits gracefully too, rather than bail with an error making our tests fail.